formatter.go

```diff
@@ -42,6 +43,9 @@ type DefaultFormatter struct {
 	// Exclude diff from failure report.
 	DisableDiffs bool
 
+	// Disable printing floats in scientific form.
```

Comment:
We should mention what happens if the flag is not set.

Reply:
ok, will update the comment

Reply:
how about?

```suggestion
	// Disable printing floats in scientific form. If not set will print floats in decimal notation.
```

Reply:
updated https://github.com/gavv/httpexpect/pull/220/commits/a8c1e6492c6083ccc1a3b858b16d711b6c43ec17

Reply:
> If not set, will print floats in decimal.

I think this is not true. If it's not set, the behavior will be to automatically switch between scientific and decimal forms, like in %g. See details in fmt docs.

Reply:
BTW if I'm right let's cover this in tests.

Reply:
Looks legit.

Currently we use %v, which uses %g, which have this in docs:

```
%g	%e for large exponents, %f otherwise. Precision is discussed below.
```

But please re-check and cover in unit test to be 100% sure :)

In our comment, we can say something like this:

* without DisableScientific, we use %g, which uses %e (scientific form) for large exponents and %f (decimal form) otherwise (see documentation for fmt package)
* with DisableScientific, we use %f (decimal form)

Reply:
BTW, after thinking about this again... :)

It seems even better and clearer API would be to replace boolean with a enum with three values:

* Auto (corresponds to %g)
* Decimal (corresponds to %f)
* Scientific (corresponds to %e)

Where Auto is equal to `0` and thus is used by default.